redis/redis/broker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM redis:latest
+
+COPY redis.conf /usr/local/etc/redis/redis.conf
```

Comment:
Was there something that needed to be customized in the redis config?  If so please add that to the readme, as the file is too long for end users to do a manual diff.  If not, please remove the config (and perhaps this dockerfile via referencing the redis:latest directly from docker compose)